From masta@perlgolf.de  Thu Aug  2 14:25:57 2007
From: Nico Erfurth <masta@perlgolf.de>
Date: Thu, 02 Aug 2007 22:25:20 +0200
Subject: USB: Add drivers/usb/misc/iowarrior.c to the Makefile
To: gregkh@suse.de
Cc: linux-kernel@vger.kernel.org, trivial@kernel.org
Message-ID: <46B23DB0.5010702@perlgolf.de>


This patch adds the new iowarrior module to the Makefile in drivers/usb.
Currently the module isn't build unless another driver from usb/misc is
selected.

Signed-off-by: Nico Erfurth <masta@perlgolf.de>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/Makefile |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/usb/Makefile
+++ b/drivers/usb/Makefile
@@ -38,6 +38,7 @@ obj-$(CONFIG_USB_EMI26)		+= misc/
 obj-$(CONFIG_USB_EMI62)		+= misc/
 obj-$(CONFIG_USB_FTDI_ELAN)	+= misc/
 obj-$(CONFIG_USB_IDMOUSE)	+= misc/
+obj-$(CONFIG_USB_IOWARRIOR)	+= misc/
 obj-$(CONFIG_USB_LCD)		+= misc/
 obj-$(CONFIG_USB_LD)		+= misc/
 obj-$(CONFIG_USB_LED)		+= misc/
